pyravendb/tests/jvm_migrated_tests/test_what_changed.py

```diff
@@ -0,0 +1,31 @@
+from pyravendb.tests.test_base import TestBase, User
+
+
+class TestWhatChanged(TestBase):
+
+    def setUp(self):
+        super(TestWhatChanged, self).setUp()
+
+    def test_has_changes(self):
+        with self.store.open_session() as session:
+            user1 = User("user1", None)
+            user2 = User("user2", 1)
+
+            session.store(user1, "users/1")
+            session.store(user2, "users/2")
+            session.save_changes()
+
+        with self.store.open_session() as session:
+            self.assertFalse(session.advanced.has_changes())
+
+            user1, user2 = session.load(["users/1", "users/2"], User)
+
+            self.assertFalse(session._has_change(user1))
```

Comment:
In the original test (in C#) it was `session.Advanced.HasChanged()`. Please add such method and replace the occurrences in these tests.

Reply:
https://github.com/ravendb/ravendb/blob/49086f551a2ec7349c0922f6ec96c189f94fb7a6/src/Raven.Client/Documents/Session/IAdvancedDocumentSessionOperations.cs#L54